api/keystore/service.go

```diff
@@ -150,13 +149,22 @@ func (ks *Keystore) ListUsers(_ *http.Request, args *struct{}, reply *ListUsersR
 	return it.Error()
 }
 
+// ExportUserArgs ...
+type ExportUserArgs struct {
+	// The username and password
+	api.UserPass
+	// The encoding for the exported user ("hex" or "cb58")
+	Encoding formatting.Encoding `json:"encoding"`
+}
+
 // ExportUserReply is the reply from ExportUser
 type ExportUserReply struct {
-	User formatting.CB58 `json:"user"`
+	// String representation of the user
+	User string `json:"user"`
```

Comment:
I think that we have a convention to have the encoding placed into the return type.